splitio/storage/pluggable.py

```diff
@@ -284,35 +288,55 @@ def get_segment_names(self):
         """
         try:
             keys = []
-            for key in self._pluggable_adapter.get_keys_by_prefix(self._prefix[:-self._SEGMENT_NAME_LEMNGTH]):
+            for key in self._pluggable_adapter.get_keys_by_prefix(self._prefix[:-self._SEGMENT_NAME_LENGTH]):
                 if key[-self._TILL_LENGTH:] != 'till':
-                    keys.append(key[len(self._prefix[:-self._SEGMENT_NAME_LEMNGTH]):])
+                    keys.append(key[len(self._prefix[:-self._SEGMENT_NAME_LENGTH]):])
             return keys
         except Exception:
             _LOGGER.error('Error getting segments')
             _LOGGER.debug('Error: ', exc_info=True)
             return None
 
-    def get_keys(self, segment_name):
+    # TODO: To be added in the future because this data is not being sent by telemetry in consumer/synchronizer mode
+#    def get_keys(self, segment_name):
+#        """
+#        Get keys of a segment.
+#
+#        :param segment_name: segment name
+#        :type segment_name: str
+#
+#        :return: list of segment keys
+#        :rtype: str[]
+#        """
+#        try:
+#            return list(self._pluggable_adapter.get(self._prefix.format(segment_name=segment_name)))
+#        except Exception:
+#            _LOGGER.error('Error getting segments keys')
+#            _LOGGER.debug('Error: ', exc_info=True)
+#            return None
+
+    def segment_contains_key(self, segment_name, key):
```

Comment:
Lets keep only `segment_contains`, and remove `segment_contains_key` for compatibility with redis

Reply:
done